src/settings/Models/FinePolicy/utils/helper.test.js

```diff
@@ -0,0 +1,21 @@
+import helpers from './helpers';
+
+describe('FinePolicy helpers', () => {
+  it('"isValidItemSelected" should return true if some matches', () => {
+    const selectedId = 'selectedId';
+    const options = [
+      { id: 'test' },
+      { id: selectedId },
+    ];
+    expect(helpers.isValidItemSelected(options, selectedId)).toEqual(true);
```

Comment:
Could you please add empty line before expect? Here and below.

Reply:
done